examples/GUI/GUIGlobe.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Generated On: 2015-10-5
+ * Class: GUIGlobe
+ * Description: Classe pour créer un menu.
+ */
+
+
+function GUIGlobe() {
+    //Constructor
+
+    this.gui = new dat.GUI( { autoPlace : false});
+    this.gui.domElement.id = 'menuDiv';
```

Comment:
Don't hardcode the element id. Add it as a parameter to the constructor instead.